typescript/commands/signature.py

```diff
@@ -60,7 +60,7 @@ def on_selected(self, index):
 
 class TypescriptSignaturePopup(sublime_plugin.TextCommand):
     def is_enabled(self):
-        return TOOLTIP_SUPPORT and is_typescript(self.view)
+        return TOOLTIP_SUPPORT and is_typescript(self.view) and get_language_service_enabled()
```

Comment:
I think it might be worthwhile to create decorators for these; basically you'd be able to do something like

* `typescript/commands/signature.py`
    ```python
    from ..libs.guards import is_typescript, language_service_enabled;

    class TypescriptSignaturePopup(sublime_plugin.TextCommand):
        @g_is_typescript
        @language_service_enabled
        def is_enabled(self):
            return TOOLTIP_SUPPORT
    ```
* `typescript/libs/guards.py`

    ```python
    def language_service_enabled(func):
        def wrapped(*args):
            func(*args) and get_language_service_enabled()
        return wrapped

    def g_is_typescript(func):
        def wrapped(self, *args):
            func(self, *args) and is_typescript()
        return wrapped
    ```